frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -380,7 +380,7 @@ type RowFilterBase<R> = {
   filterGroupName: string;
   type: string;
   items: RowFilterItem[];
-  filter: (input: FilterValue, obj: R) => boolean;
+  filter?: (input: FilterValue, obj: R) => boolean;
```

Comment:
this has to stay required as `filter` is the actual filtering function to be applied on the data